<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="backgroundwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{ 
background-color: qlineargradient(spread:pad, x1:0.384211, y1:0.023, x2:0.768, y2:1, stop:0 rgba(29, 30, 73, 255), stop:1 rgba(39, 41, 100, 255));}</string>
   </property>
   <widget class="QLabel" name="backgroundPhoto">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>900</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Media/background.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="resnetButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 15px;
	background-color: qlineargradient(spread:pad, x1:0.384211, y1:0.023, x2:0.768, 		y2:1, stop:0 rgba(29, 30, 73, 255), stop:1 rgba(39, 41, 100, 255));

	font: 75 14pt &quot;Comic Sans MS&quot;; color: rgb(110, 164, 190)
}

QPushButton:hover{
	background-color: rgb(54, 57, 138);
}
QPushButton:pressed{
	background-color: rgb(91, 96, 230);
}</string>
    </property>
    <property name="text">
     <string>ResNet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FACButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 15px;
	background-color: qlineargradient(spread:pad, x1:0.384211, y1:0.023, x2:0.768, 		y2:1, stop:0 rgba(29, 30, 73, 255), stop:1 rgba(39, 41, 100, 255));

	font: 75 22pt &quot;Comic Sans MS&quot;; color: rgb(110, 164, 190)
}

QPushButton:hover{
	background-color: rgb(54, 57, 138);
}
QPushButton:pressed{
	background-color: rgb(91, 96, 230);
}</string>
    </property>
    <property name="text">
     <string>Football Action
Classification</string>
    </property>
   </widget>
   <widget class="QLabel" name="howItWorksText">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Comic Sans MS&quot;; color: rgb(110, 164, 190)</string>
    </property>
    <property name="text">
     <string>Please choose a model</string>
    </property>
   </widget>
   <widget class="QLabel" name="howItWorksHeadText">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;Comic Sans MS&quot;; color: rgb(110, 164, 190);
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>TRAINING BY PICTURE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decisionTreeButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>220</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 15px;
	background-color: qlineargradient(spread:pad, x1:0.384211, y1:0.023, x2:0.768, 		y2:1, stop:0 rgba(29, 30, 73, 255), stop:1 rgba(39, 41, 100, 255));

	font: 75 13pt &quot;Comic Sans MS&quot;; color: rgb(110, 164, 190)
}

QPushButton:hover{
	background-color: rgb(54, 57, 138);
}
QPushButton:pressed{
	background-color: rgb(91, 96, 230);
}</string>
    </property>
    <property name="text">
     <string>SomethingNet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="predictButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>520</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 15px;
	background-color: qlineargradient(spread:pad, x1:0.384211, y1:0.023, x2:0.768, 		y2:1, stop:0 rgba(29, 30, 73, 255), stop:1 rgba(39, 41, 100, 255));

	font: 75 13pt &quot;Comic Sans MS&quot;; color: rgb(110, 164, 190)
}

QPushButton:hover{
	background-color: rgb(54, 57, 138);
}
QPushButton:pressed{
	background-color: rgb(91, 96, 230);
}</string>
    </property>
    <property name="text">
     <string>Predict</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goBackButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 15px;
	background-color: qlineargradient(spread:pad, x1:0.384211, y1:0.023, x2:0.768, 		y2:1, stop:0 rgba(29, 30, 73, 255), stop:1 rgba(39, 41, 100, 255));

	font: 75 13pt &quot;Comic Sans MS&quot;; color: rgb(110, 164, 190)
}

QPushButton:hover{
	background-color: rgb(54, 57, 138);
}
QPushButton:pressed{
	background-color: rgb(91, 96, 230);
}</string>
    </property>
    <property name="text">
     <string>GoBack</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>validation size</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Learning rate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="background.qrc"/>
 </resources>
 <connections/>
</ui>
